http/http_client.hpp

```diff
@@ -149,9 +149,10 @@ class ConnectionInfo : public std::enable_shared_from_this<ConnectionInfo>
     // Ascync callables
     std::function<void(bool, uint32_t, Response&)> callback;
     crow::async_resolve::Resolver resolver;
-    boost::asio::ip::tcp::socket conn;
+    boost::asio::io_context& ioc;
     std::optional<boost::beast::ssl_stream<boost::asio::ip::tcp::socket&>>
         sslConn;
+    std::unique_ptr<boost::asio::ip::tcp::socket> conn;
```

Comment:
Why did you need to move this to a pointer?

Reply:
because we are not  planning to reuse the socket.  I don't know what benefit we get in reuse. But there are troubles seen after keep-alive time out happens.

Reply:
Ack